glslc/src/file_compiler.h

```diff
@@ -99,13 +96,26 @@ class FileCompiler {
   // compiler_.CompileGlslToSpv().
   shaderc::CompileOptions options_;
 
-  // Returns the name of the output file, given the input_filename string.
+  // Returns the final file name to be used for the output file.
+  // 
+  // If an output file name is specified by the argument of -o option, use that
```

Comment:
`FileCompiler` knows nothing about `-o`.  I think you meant `SetOutputFileName()`?